modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -483,6 +483,32 @@ def test_astype():
     with pytest.raises(KeyError):
         modin_df.astype({"not_exists": np.uint8})
 
+    # The dtypes series must have a unique index.
```

Comment:
If pandas has similar behavior, we should be testing both Modin and pandas via eval_general.

Reply:
Done.